Readme.md

```diff
@@ -1,176 +1,341 @@
+**If you are a Developer head to [Getting Started](#GettingStarted)**
+
+**If you are a Designer, Marketer, or Investor**
+    Please Email Oliver Sauter: oli@worldbrain.io 
+    
+**If you would like to donate a bit of money**
+Please support us on [Patreon](https://www.patreon.com/WorldBrain)
+
+### <a id="getting-started" href="#getting-started">Getting Started</a>
+
+We have broken up tasks into three levels, **easy**, **medium** and **hard** tasks. You can choose based on your skill level or how much time you would like to put into contributing. You can also check out [Issues](issues) to see what bugs need to be squashed. **After finding a Task you would like to help out with please see [Installation Instructions](#Installation)**.   
+
+### Easy
+**TODO**
+### Medium
+**TODO**
+### Hard
+**TODO**
+
+### <a id="installation" href="#installation">Installation</a>
+
+**This assumes a basic knowledge of `git`, `npm` and usage of the `command line`, if you are uncomfortable with these tools please see *[the basics]***
+
+### First steps:
+**Clone this repo:**
+
+```sh
+$ git clone https://github.com/WorldBrain/WebMemex
+```
+
+**Install yarn:** 
+```sh
+$ npm install -g yarn
+```
+
+**Run `yarn` to install dependencies**
+```sh
+$ yarn
+```
+**This could take a while....**
+
+**Now run `yarn watch` to compile incremental builds**
+```sh
+$ yarn watch
+```
+
+**For Windows Users! Please run: **TODO**** 
+```sh
+$ yarn watch **TODO**
+```
+
+### <a id="running-the-extension" href="#running-the-extension">Running The Extension:</a>
+
+As of now it should work in most modern browsers except Safari (we mainly use [Chrome](#Chrome) for testing so if any inconsistencies are found in [Firefox](#Firefox), Opera or any other browser please create an [issue][issue] and submit a fix)
+
+**Note: It is highly recommended to [create a new user](#Creatinganewuser) for dev purposes especially if you currently use the extension and would like to develop without interfering in it's daily use**
+
+### Creating a New User
+*Chrome:*
+1. In the top right corner of your browser click the user icon (above the menu). Alternatively go into [settings](chrome://settings/)
+2. Now click **Manage People**, in settings: Under *People* click **Manage other people**
+3. Now click **Add People** in the lower left
+4. Choose a name such as **Worldbrain Test** and any icon. 
+5. All Finished! :tada:
+
+*Firefox:*
+1. In firefox url type in [about:profiles](about:profiles)
+2. Click `Create a new Profile`
+3. Hit `continue`
+4. Now enter a name such as **Worldbrain Test** and hit `continue`
+5. You should see the new Profile and can now click `Launch profile in new browser`
+6. All Finished! :tada: For more info see: [creating multiple profiles](https://developer.mozilla.org/en-US/Firefox/Multiple_profiles)
+
+### Running + Debugging
+*Chrome*
+1. Open a `New Tab`
+2. Click on the `Menu`
+3. Go to `More Tools`
+4. Click `Extensions`

```

Comment:
I think steps 2 - 4 can just be replaced with:
"Enter `about:extensions` into the address bar"
like with the FF instructions